packages/ssz/test/util/testCases.ts

```diff
@@ -1 +1,57 @@
-export const TEST_CASE_LOCATION = "../../../../../../node_modules/@chainsafe/eth2-spec-tests";
\ No newline at end of file
+import {describeDirectorySpecTest, InputType, safeType} from "@chainsafe/eth2.0-spec-test-util";
```

Comment:
Isn't this file duplicate of /test/spec/util?

Reply:
I don't think so, this is a templated function for running the "static" ssz spec tests

Reply:
I'm pretty sure they are identical:
packages/ssz/test/spec/util.ts

Reply:
doh, yeah, will remove duplicate